setup.cfg

```diff
@@ -1,8 +1,2 @@
-[bdist_wheel]
-# This flag says that the code is written to work on both Python 2 and Python
-# 3. If at all possible, it is good practice to do this. If you cannot, you
-# will need to generate wheels for each Python version that you support.
-universal=1
-
 [aliases]
-test=pytest  # Use pytest to run tests.
+test=pytest
```

Comment:
Does ``python setup.py test`` still work without the test runner? Otherwise we could remove the whole file

Reply:
It doesn't...

Reply:
...but for other reasons. Using `python setup.py test` fails in the environment test:
```
____________________________________________ ERROR collecting typhon/tests/test_environment.py _____________________________________________
ImportError while importing test module '/Users/u237002/Hacking/git/typhon/typhon/tests/test_environment.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
typhon/tests/test_environment.py:9: in <module>
    from typhon import environ
E   ImportError: cannot import name 'environ' from 'typhon' (/Users/u237002/Hacking/git/typhon/typhon/__init__.py)
```